### ISSUE TITLE
Modal

### ISSUE THREAD

**Issue by karigaron17** [hlxsites/maidenform#52]
Modal trigger in promo banner does not state it has an expandable region.
Modal once open does not announce it has bene opened and is expanded.
Modal when open does not trap keyboard user into the modal window allowing the site to be tabbed with modal open.
Modal cannot be closed with 'esc'.

See for more info:
https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/
